packages/gibsonify_api/lib/src/gibsonify_api.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:convert';
+
+import 'package:gibsonify_api/gibsonify_api.dart';
+
+/// API that handles requests for Gibsons Forms and Recipes. Provides
+/// functionality for loading and saving Forms and Recipes to local storage by
+/// using the SharedPreferences plugin.
+class GibsonifyApi {
+  final SharedPreferences _sharedPreferences;
+  GibsonifyApi({required SharedPreferences sharedPreferences})
+      : _sharedPreferences = sharedPreferences;
+
+  static const gibsonFormsKey = 'gibsonify_gibsons_forms_key';
+
+  /// Attempts to load a list Forms from shared preferences local storage.
+  /// Returns an empty list if no data is found or data cannot be decoded.
+  List<GibsonsForm?> loadForms() {
+    List<GibsonsForm?> gibsonsForms = [];
+    String? gibsonsFormsRaw = _sharedPreferences.getString(gibsonFormsKey);
+    if (gibsonsFormsRaw == null || gibsonsFormsRaw.isEmpty == true) {
+      return [];
+    } else {
+      try {
+        List<dynamic> partiallyDecodedGibsonsForms =
+            jsonDecode(gibsonsFormsRaw);
+        gibsonsForms = partiallyDecodedGibsonsForms
+            .map((e) => GibsonsForm.fromJson(e))
+            .toList();
+      } catch (e) {
+        return [];
+      }
+      return gibsonsForms;
+    }
+  }
```

Comment:
This implementation looks great and seems to perform as intended - I assume you're going to implement a similar methods for recipe after merging the recipe PR?